config_defaults/subtests/docker_cli/systemd.ini

```diff
@@ -1,8 +1,17 @@
 [docker_cli/systemd]
-docker_timeout = 300
+subsubtests = systemd_pull, systemd_build , systemd_run
+
 #: Systemd unit file
+sysd_unit_file = docker-pull.service
+#: image name to pull, build, run
+image_name = rhel7:latest
```

Comment:
I forget, is the image used here is also referenced in ``FROM`` to build that ``4321`` test image?  IMHO, it would be good to get them all in sync, but that probably means updating the dockerfile's ``FROM`` line.  We do this in the ``build`` test.

hmmm

Yeah, let's replace them all with the CentOS base image.  That's (more) guaranteed to work forever & wherever.  I'm fine with having it just use ``image_name`` for the foreseeable future, though it would be a nice update to make it work with the conventional mechanisms.  It's more important that the systemd feature get tested than all the niggly details are right from the start.